### ISSUE TITLE
Uncaught TypeError: Cannot read property 'pomPath' of undefined

### ISSUE THREAD

**Issue by theskyinflames** [concon121/atom-maven#42]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.7.4
**System**: Ubuntu 14.04.4
**Thrown From**: [atom-maven](https://github.com/concon121/atom-maven) package, v0.12.1
### Stack Trace

Uncaught TypeError: Cannot read property 'pomPath' of undefined

```
At /home/jaumearus/.atom/packages/atom-maven/lib/atom-maven.js:23

TypeError: Cannot read property 'pomPath' of undefined
    at /home/jaumearus/.atom/packages/atom-maven/lib/atom-maven.js:23:34
    at setTimeout (/home/jaumearus/.atom/packages/atom-maven/lib/pom-factory.js:31:3)
```
### Commands

```
     -0:19.2.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.seti-ui-no-icons.theme-one-light-syntax.theme-one-light-ui)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-maven, v0.12.1
autocomplete-go, v1.1.0
build-dockerfile, v1.0.1
builder-go, v1.0.0
dockerletion, v0.2.0
environment, v1.2.0
file-icons, v1.7.13
go-config, v1.1.4
go-debug, v0.1.12
go-get, v1.0.3
go-plus, v4.1.0
go-rename, v0.3.0
gofmt, v1.1.6
gometalinter-linter, v1.1.0
gorename, v1.0.2
highlight-line, v0.11.1
language-docker, v1.1.6
language-protobuf, v0.5.0
linter, v1.11.4
linter-docker, v0.1.2
linter-protocol-buffer, v0.2.0
navigator-godef, v1.0.3
tester-go, v1.0.4

# Dev
No dev packages
```

**Comment by concon121** [comment]
Would it be possible to get a sample pom structure to reproduce the issue?

**Comment by concon121** [comment]
Added a semi-fix in 0.14.0, which doesn't necessarily fix the underlying problem but should stop the exception being thrown. 

Still require info on how to reproduce this issue.
